### PR TITLE
Tag stages take two - additional tags for the rootless install - replacement PR

### PR DESCRIPTION
**Pull request by thecrazymonkey**
# Description

This is a replacement for the original PR 639, merged with 6.2.x code and review changes applied. I have no permissions to the original PR so created a new one. 

Original description:

Add additional tags to various stages, allowing certain operations to be skipped if they require root access (using --skip-tags).

Adds additional variables that does not hardcode the location of /etc/, so that configuration files can be placed in alternative locations (applies mostly to archive installation).

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

Completed archive based installation with and without specifying --skip-tags.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

File: roles/kafka_broker/tasks/set_principal.yml [hunk 1]
```diff
@@ -26,8 +26,8 @@
   # Extract DNAME from line
   # Remove spaces after commas
   shell: |
-    keytool -list -keystore {{kafka_broker_keystore_path}} \
-        -storepass {{kafka_broker_keystore_storepass}} -v \
+    keytool -list -keystore {{kb_keystore_path}} \
```
[the remark this answers is not in the document]
Comment (thecrazymonkey) [reply roles/kafka_broker/tasks/set_principal.yml:29]: This is to cater for customer deployment where each broker has different keystore and keystore password, this is related to change in rbac.yml where the variables are set specifically from the host running this code

File: VARIABLES.md [hunk 2]
```diff
@@ -1484,6 +1526,14 @@ Default:  ""
 
 ***
 
+### ksql_config_prefix
+
+Default KSQLDB config prefix
+
+Default:  "{{ config_prefix }}/{{(confluent_package_version is version('5.5.0', '>=')) | ternary('ksqldb' , 'ksql')}}"
```
[the remark this answers is not in the document]
Comment (thecrazymonkey) [reply VARIABLES.md:1533]: Good point, will modify

File: roles/kafka_broker/tasks/set_principal.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (nsharma-git) [reply roles/kafka_broker/tasks/set_principal.yml:29]: Thanks

**Review by nsharma-git — APPROVED** [review]

**Comment by nsharma-git** [comment]
This is an old PR and corresponding Jenkins pre-check has expired. I have verified it on my setup and it works fine.

**Pull request merged**